app/src/main/java/com/example/wearit/ui/WardrobeScreen.kt

```diff
@@ -355,13 +409,13 @@ fun WardrobeNavigationSection(
 
             MasterButton(
                 type = ButtonType.WHITE,
-                onClick = { /*todo*/ },
+                onClick = { toggleEdit(editing) },
```

Comment:
To move być po prostu: onClick = toggleEdit albo onClick = { toggleEdit() }.
(Patrz komentarz wyżej)

Reply:
oki, zmienię